styles/venture/layout.s2

```diff
@@ -1217,118 +1228,138 @@ $module_layout_css
 .one-column #content .module-content { text-align: left;}
 
 .two-columns-left #content,
-.three-columns-sides #content { border-left: $*sidebar_width solid transparent; }
+.three-columns-sides #content { border: none; float: none; }
 
-.two-columns-right #content,
-.three-columns-sides #content { border-right: $*sidebar_width solid transparent; }
+.column-right #content { min-width: $*sidebar_width; box-sizing: border-box; }
 
-.three-columns-left #content { border-left: $*sidebar_width_doubled solid transparent; }
+#content { padding: 1em; }
 
-.three-columns-right #content { border-right: $*sidebar_width_doubled solid transparent; }
+@media $medium_media_query {
+    .two-columns-left #content,
+    .three-columns-sides #content { border-left: $*sidebar_width solid transparent; border-right: $*sidebar_width solid transparent; }
 
-.column-right #content { min-width: $*sidebar_width; }
+    .three-columns-left #content { border-left: $*sidebar_width_doubled solid transparent; }
+
+    .two-columns-right #content,
+    .three-columns-right #content { border-right: $*sidebar_width_doubled solid transparent; }
+
+    .column-right #content { box-sizing: content-box; }
+}
 
```

Comment:
I think something has gotten kind of off here, possibly because Venture's existing CSS seems like it might be wonky? The outer margins don't seem to be applying properly on any layout on desktop, and the two column versions have way too much space on the right-hand side of the page

Reply:
wtf??? can you check http://momijizukamori.momiji.hack.dreamwidth.net/ and see if you get the same thing? I'm seeing it on both my 1920x1200 monitor and laptop (in both FF and Chrome), and I'm pretty sure I was looking on my other laptop originally. When I shrink the page enough to flip it into one-column mode it looks fine but before that, no.

Reply:
okay there is *something* up here because it does in fact look like it does in your screencap on your dreamhack but it still looks like *my* screencap on *my* dreamhack

what. what is happening. is the computer haunted

Reply:
okay it looks like the styles are in fact different between the two dreamhack instances so mmmmaybe i'll recompile styles once i'm back on my personal machine and in the meantime when you get the chance recompile yours and see if anything changes???

Reply:
ok never mind i was the fool who needed to recompile. fix incoming shortly